src/setupTest.js

```diff
@@ -45,7 +45,7 @@ export function initializeMockApp() {
 
 let globalStore;
 
-export async function initializeTestStore(preloadedState = null, overrideStore = true) {
```

Comment:
I'm not sure why this was marked `async`. There's nothing asynchronous about it. It might have been copy pasted from `frontend-app-learning`'s `initializeTestStore` function, which does have asynchronicity.